xpublish_wms/wms/get_map.py

```diff
@@ -204,18 +204,15 @@ def select_time(self, da: xr.DataArray) -> xr.DataArray:
         :param da:
         :return:
         """
-        time_dim = da.cf.coordinates.get(self.TIME_CF_NAME, None)
-        if time_dim is not None and len(time_dim):
-            time_dim = time_dim[0]
-
-        if not time_dim or time_dim not in list(da.dims):
-            return da
-
-        if self.time is not None:
-            da = da.cf.sel({self.TIME_CF_NAME: self.time}, method="nearest")
-        elif self.TIME_CF_NAME in da.cf.coords:
-            da = da.cf.isel({self.TIME_CF_NAME: -1})
-
+        # by using coords, we will fallback to ds.coords[self.TIME_CF_NAME]
```

Comment:
the real issue is that `coordinates` was used here but `coords` is used in `gridded.additional_coordinates` so a `time` variable that doesn't have the right cf-attributes is skipped here but ALSO not added to `dim_selectors`.